### PR TITLE
Feature/instruction

### PR DESCRIPTION
**Pull request by TOSINNIJIS1**
# Description
Instruction page done and added an onClick for the cancel button
https://trello.com/c/QtCMcH9U/39-instruction-page-popup-card-2
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -52,6 +53,7 @@ const App = () => {
         <Route path="/upload" component={UploadForm} />
         <Route path="/upload-save" component={UploadSudoku2} />
         <Route path="/about" component={TeamList} />
+        <Route path="/soon" component={SoonPage} />
```
Comment (aanderson9313) [inline src/App.js:56]: i'm assuming you just went ahead and added the route while you were working on the instruction page and that is why there is more than one PR on here?

Comment (malexmad) [reply src/App.js:56]: Looking at this PR and the other [PR](https://github.com/Lambda-School-Labs/omega2020-fe/pull/88/commits/6c8df3432920120392a618f081dc9f58430bd20a)
 changes, they have they same file changes other then the 2 extra files for instructions

Comment (aanderson9313) [reply src/App.js:56]: so do you want him to create a whole new pr on the other branch then?

Comment (malexmad) [reply src/App.js]: ok merging the other PR to master, then pulling changes from master resolved the issue I was seeing. Good to merge

**Pull request merged**